client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx

```diff
@@ -445,10 +479,47 @@ function StatsCommercialFlowOptOutForm( {
 				</ul>
 			</div>
 			<div className={ `${ COMPONENT_CLASS_NAME }__personal-checklist-button` }>
-				<Button variant="secondary" disabled={ isFormSubmissionDisabled() } onClick={ formHandler }>
-					{ formButton }
-				</Button>
+				{ supportsOnDemandCommercialClassification && (
+					<Button
+						variant="secondary"
+						disabled={ hasRunLessThan3DAgo || ! allConditionsChecked }
+						onClick={ handleCommercialClassification }
+					>
+						{ translate( 'Reverify' ) }
+					</Button>
+				) }
+				{ ( ! supportsOnDemandCommercialClassification ||
+					( ! isClassificationInProgress && commercialClassificationLastRunAt > 0 ) ) && (
+					<Button
+						variant="secondary"
+						disabled={ ! supportsOnDemandCommercialClassification && isFormSubmissionDisabled() }
+						onClick={ formHandler }
+					>
+						{ formButton }
+					</Button>
+				) }
 			</div>
+			{ supportsOnDemandCommercialClassification && (
+				<>
+					{ errorMessage && (
+						<p className={ `${ COMPONENT_CLASS_NAME }__error-msg` }>Error: { errorMessage }</p>
```

Comment:
I think we should pass the message without indicating it as an error to users and handle it internally if it is exactly an error.

```suggestion
						<p className={ `${ COMPONENT_CLASS_NAME }__error-msg` }>{ errorMessage }</p>
```

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/88931/commits/0034940c8d7d9019b825bd78ae25079ba5fcbc44.

Reply:
But indeed, there would be some unexpected system errors we need to handle. 🤔

Reply:
I agree. Thanks!